frontend/js/src/routes/routes.tsx

```diff
@@ -17,7 +17,10 @@ import getPlayerRoutes from "../player/routes";
 import getRecommendationsRoutes from "../recommended/tracks/routes";
 import getPlayingNowRoutes from "../player/routes/listening-now-routes";
 
-const getRoutes = (musicbrainzID?: string): RouteObject[] => {
+const getRoutes = (
+  musicbrainzID?: string,
+  withBrainzPlayer?: boolean
```

Comment:
Can't comment on the appropriate line, so commenting here. Your modification on line 60 only affects private routes (settings, feed, etc.).
You also need to set `withBrainzPlayer={withBrainzPlayer}` on line 42 for the `element` layout component